code/modules/mob/mob.dm

```diff
@@ -254,6 +254,11 @@
 			//Now, S represents a container we can insert W into.
 			S.handle_item_insertion(W, TRUE, TRUE)
 			return S
+		if(istype(back, /obj/item/mod/control))
```

Comment:
you should consider turning this istype check into a define since you use it so much

Reply:
Done!